src/api/v1/community_businesses/schema.ts

```diff
@@ -39,4 +39,16 @@ export const visitActivitiesPutPayload = {
   sunday: Joi.boolean(),
 };
 
+export const cbPayload = {
+  name: Joi.string().min(1),
+  region: Joi.string().min(1),
+  sector: Joi.string().min(1),
+  logoUrl: Joi.string().uri(),
+  address1: Joi.string().min(1),
+  address2: Joi.string().min(1),
+  townCity: Joi.string().min(1),
+  postCode: Joi.string().min(6).max(10),
+  turnoverBand: Joi.string().min(5).max(11),
+  _360GivingId: Joi.string().min(5).max(11), };
```

Comment:
Where does the upper limit of 11 come from? From what I can see [there's no guarantee](http://standard.threesixtygiving.org/en/latest/identifiers/#) that IDs won't be longer than 11 characters. IMO let's make it the same as the DB constraint (255), or just remove the upper limit.

Reply:
addressed in #327